inter,inter-ui: keep font name the same for attributes, split off

dtzWill opened the pull request:
Feedback please! This is one solution to the problem,
but since it's a small amount of work wanted an actual PR
and so if we like it it's easy to proceed.

###### Motivation for this change

https://github.com/NixOS/nixpkgs/pull/57605#issuecomment-477874093

From upstream repository activity, it appears they're working
to be included in google fonts?
However I'm not sure it's worth "waiting" on that,
and it's nice to have the font you wnt directly sometimes
instead of the big bundle :).

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

dtzWill committed:
Revert "inter-ui: 3.1 -> 3.3 (#57605)"

Don't change font name for the `inter-ui` attribute,
breaks configs (as reported by a user on the PR).

This reverts commit 4d82af02503fec564ba71453ea53a14ff72adbc5.

dtzWill committed:
inter,inter-ui: init new 'inter', bump inter-ui to last w/that name

ivan commented:
Thanks! I tested this with `pkgs.inter` in my `fonts.fonts` and it seems to be working.

dtzWill committed:
Revert "inter(-ui): 3.3 -> 3.4"

This reverts commit 4d7a9732336aeffa2772d2840598eb3e8ef599cf.

dtzWill committed:
inter: 3.3 -> 3.5

https://github.com/rsms/inter/releases/tag/v3.5
https://github.com/rsms/inter/releases/tag/v3.4

dtzWill commented:
@GrahamcOfBorg build inter inter-ui

dtzWill commented:
Merging before it becomes even more messy to do so :).
